### PR TITLE
RA-1870 : Appointment Scheduling Fields are Misaligned

### PR DESCRIPTION
**Pull request by NehalJain159**
RA-1870 : Appointment Scheduling Fields are Misaligned
Link of the issue : https://issues.openmrs.org/browse/RA-1870
Modified the ui to combat the problem of overwriting. 

Please do let me know your reviews @gracepotma @sherrif10 and suggest amendments if needed.
Big-Screen
MediumScreen

### CODE REVIEW

**Comment by NehalJain159** [comment]
I was unable to figure out how to work on different versions and haven't done the same in this issue.Any help in this regard would be appreciated.

Thanks!

File: omod/src/main/compass/sass/scheduleAppointment.scss [hunk 1]
```diff
@@ -3,7 +3,7 @@
 }
 
 .inlineBox {
-  float: left;
+  display:inline-block;
```
Comment (sherrif10) [inline omod/src/main/compass/sass/scheduleAppointment.scss:6]: Wow thanks for working on this, you have made it more simpler than i had  done cc @dkayiwa  , this LGTM

**Review by sherrif10 — APPROVED** [review]

File: omod/src/main/webapp/fragments/scheduleAppointment.gsp [hunk 2]
```diff
@@ -93,7 +93,7 @@ ${ ui.includeFragment("appointmentschedulingui", "timeZoneWarning") }
                </daterangepicker>
 
                 <% if (canOverbook) { %>
-                   <div id="selectIncludeSlotsThatRequireOverbook" class="inlineBox">
+                   <div class="inlineBox">
```
Comment (HerbertYiga) [inline omod/src/main/webapp/fragments/scheduleAppointment.gsp:96]: thanks @NehalJain159  how do things appear on minimizing and maximizing the page

File: omod/src/main/compass/sass/scheduleAppointment.scss (hunk 1 above)
Comment (NehalJain159) [reply omod/src/main/compass/sass/scheduleAppointment.scss:6]: Thanks for your review!

File: omod/src/main/webapp/fragments/scheduleAppointment.gsp (hunk 2 above)
Comment (NehalJain159) [reply omod/src/main/webapp/fragments/scheduleAppointment.gsp:96]: @HerbertYiga ,I have added the ss for big and medium screeen respectively as the 2 attachments in the initial pullRequest.Could you please review it and for small screen the ss is atached below
SmallScreen

**Comment by NehalJain159** [comment]
Please review @dkayiwa

**Comment by sherrif10** [comment]
can you follow this approach like i had done it here https://github.com/openmrs/openmrs-module-appointmentschedulingui/pull/18

File: omod/src/main/compass/sass/scheduleAppointment.scss (hunk 1 above)
Comment (brandones) [reply omod/src/main/compass/sass/scheduleAppointment.scss:6]:
```suggestion
  display: inline-block;
```

**Review by brandones — APPROVED** [review]
Thanks @NehalJain159 ! Looks good to me.

**Pull request merged**